rspec-mode.el

```diff
@@ -273,6 +273,12 @@ They execute after failures have been stored in `rspec-last-failed-specs'."
   :type 'hook
   :group 'rspec-mode)
 
+(defcustom rspec-allow-multiple-compilation-buffers nil
```

Comment:
1. Naming is hard! Happy to accept alternatives
2. Since this broke at least one other person's workflow (see https://github.com/pezra/rspec-mode/issues/180#issuecomment-442123722), I'm electing to make this off by default. If you feel strongly otherwise, I'm happy to change this.

Reply:
I'd probably call it `rspec-use-per-project-buffers`, but then again, it seems I'm not entirely clear on the idea behind the feature (the buffer name function is not deterministic).

/Cc @pezra @kzkn

Reply:
IIRC, this feature was intended to allow running logically different verifications simultaneously. For example, verifying multiple different files in the same project, verifying a single file and its whole project, or two separate projects. 

I don't think "per-project" is quite right but there are limits to the multiple simultaneous compilations allowed.